python/docs/source/index.rst

```diff
@@ -21,8 +21,42 @@
 PySpark Documentation
 =====================
 
+PySpark is an interface for Apache Spark in Python language. It not only offers for you
+to write an application in the Python APIs but also provides PySpark shell so you can
```

Comment:
Maybe "It not only allows you to write Spark applications using Python APIs, but also provides the PySpark shell for interactively analyzing ..."